### PR TITLE
AKA-1 Create Customer tests

### PR DESCRIPTION
**Pull request by Kyrevam**
It includes the test cases for Creating a New Customer in Akaunting platform

### CODE REVIEW

File: .gitignore
Comment (martinhel2001) [inline .gitignore:1]: This file should not be included in the PR

File: Automation-Campus.iml
Comment (martinhel2001) [inline Automation-Campus.iml:1]: This file should not be included in the PR

File: Logs/Webliv_Automation_Logs.log
Comment (martinhel2001) [inline Logs/Webliv_Automation_Logs.log:1]: This file should not be included in the PR

File: extent-reports/extent-report.html
Comment (martinhel2001) [inline extent-reports/extent-report.html:1]: This file should not be included in the PR

File: target/classes/baseMain/TestsConfigReader.class
Comment (martinhel2001) [inline target/classes/baseMain/TestsConfigReader.class:1]: This file should not be included in the PR

File: src/main/java/dataentities/akaunting/Customer.java [hunk 1]
```diff
@@ -1,13 +1,16 @@
...
 public class Customer {
-    private String name ="";
-    private String Address="123 Elm Street";
-    private String city="Toronto";
-    private String state="TO";
-    private String country="Canada";
-    private String currency = "Argentine Peso";
-    private String Email;
+    @Setter @Getter private String name ="";
+    @Setter @Getter private String Address="123 Elm Street";
```
Comment (martinhel2001) [inline src/main/java/dataentities/akaunting/Customer.java:8]: The @Setter is not needed

**Review by martinhel2001 — CHANGES_REQUESTED** [review]
Guille pls verify all my comments

File: .gitignore
Comment (Kyrevam) [reply .gitignore:1]: This is done.

File: Automation-Campus.iml
Comment (Kyrevam) [reply Automation-Campus.iml:1]: This is done.

File: extent-reports/extent-report.html
Comment (Kyrevam) [reply extent-reports/extent-report.html:1]: This is done.

File: target/classes/baseMain/TestsConfigReader.class
Comment (Kyrevam) [reply target/classes/baseMain/TestsConfigReader.class:1]: This is done.

File: src/main/java/dataentities/akaunting/Customer.java (hunk 1 above)
Comment (Kyrevam) [reply src/main/java/dataentities/akaunting/Customer.java]: This is done.

**Review by martinhel2001 — APPROVED** [review]

**Pull request merged**